distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -833,23 +849,112 @@ public void testCswQueryByTitle() throws Exception {
 
     @Test
     public void testCswQueryForMetacardXml() throws Exception {
-        String titleQuery = getCswQuery("title",
-                "myTitle",
-                "application/xml",
-                "urn:catalog:metacard");
 
-        given().contentType(ContentType.XML)
-                .body(titleQuery)
-                .when()
-                .post(CSW_PATH.getUrl())
-                .then()
-                .assertThat()
-                .body(hasXPath("/GetRecordsResponse/SearchResults/metacard/@id",
-                        is(metacardIds[GEOJSON_RECORD_INDEX])),
-                        hasXPath("/GetRecordsResponse/SearchResults/@numberOfRecordsReturned",
-                                is("1")),
-                        hasXPath("/GetRecordsResponse/SearchResults/@recordSchema",
-                                is("urn:catalog:metacard")));
+        // The first few bytes of the metacard's base-64 encoded thumbnail image
+        String thumbNailBase64EncSubstring = "/9j/4AAQSkZJRgABAQAAAQABAAD";
+        String metacardTitle = "myTitle";
+        String extractXmlRegex = ".*?(<metacard.*?</metacard>).*";
+
+        // Define the assertions that test the query results conform to expectations
+        List<Matcher<?>> assertions = getXpathMatchers(METACARD_URI, 1);
```

Comment:
These assertion verify the result from the CWS query. We want to make sure the query result it good before we try to transform it, or else subsequent assertions will fail. Those failures would mask the real source of the problem-- the query results.